src/e2e-test/java/io/cdap/plugin/utils/SalesforceClient.java

```diff
@@ -71,9 +75,10 @@ public class SalesforceClient {
   private static final String REST_ENDPOINT = PluginPropertyUtils.pluginProp("rest.api.endpoint");
   private static final String API_VERSION = PluginPropertyUtils.pluginProp("rest.api.version");
   private static final Header prettyPrintHeader = new BasicHeader("X-PrettyPrint", "1");
+  public static List<JsonObject> objectResponseList = new ArrayList<>();
+  public static String id;
```

Comment:
update the name here as well id -> uniqueRecordId

Reply:
done